python/templates/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Language detector</title>
+  <link rel='stylesheet' href='/static/stylesheets/style.css' />
+</head>
+<body>
+  <h1>Language detector</h1>
+  <form action="/" method="get">
+    <input type="text" name="text" value="{{ text }}"/>
+    <button type="submit">Detect</button>
+  </form>
+  <br />
+  {% if error %}
+    <div>Error: {{ error }}</div>
+  {% endif %}
+  <br />
+  <div>
+    {% if language %}
+      <div>Detected {{ language }}</div>
+      <div>Score <b>{{ score }}</b> means <b>{{ scoreText }}</b></div>
+      <div>Text `{{ text }}`</div>
+    {% endif %}
+  </div>
+  <hr>
+  {% if cs_result %}
+    <div style="color: white"> {{ cs_result }}</div>
```

Comment:
I havent run this, but does white color show up? I don't see a black BG

Reply:
It's not supposed to show up unless you highlight it in the browser. Weird., but consistent with apps for the other languages.

Reply:
Huh. I feel like a dl/dt expando box would be better experience. But yeah if its consistent, carry on!

Reply:
i hear you but consistency is king :)